README.md

```diff
@@ -55,230 +57,82 @@ Log4j
-* `public String stringVariation(String key, Target target, String defaultValue)`
-* `public double numberVariation(String key, Target target, int defaultValue)`
-* `public JsonObject jsonVariation(String key, Target target, JsonObject defaultValue)`
-
-react on event:
-* `public void on(@NonNull Event event, @NonNull Consumer<String> consumer)`
-* `public void off()`
-* `public void off(@NonNull Event event)`
-* `public void off(@NonNull Event event, @NonNull Consumer<String> consumer)`
-
-manual update (webpush):
-* `public void update(@NonNull Message message)`
-
-close SDK:
-* `public void destroy()` @deprecated
-* `public void close()`
-
-## Fetch evaluation's value
-
-It is possible to fetch a value for a given evaluation. Evaluation is performed based on a different type. In case there
-is no evaluation with provided id, the default value is returned.
-
-Use the appropriate method to fetch the desired Evaluation of a certain type.
-
-### Bool variation
-
-```
-boolean result = cfClient.boolVariation("sample_boolean_flag", target, false);  
-```
-
-### Number variation
-
-```
-double result = cfClient.numberVariation("sample_number_flag", target, 0);  
-```
-
-### String variation
-
-```
-boolean result = cfClient.stringVariation("sample_string_flag", target, "");  
-```
+This quickstart assumes you have followed the instructions to [setup a Feature Flag project and have created a flag called `harnessappdemodarkmode` and created a server API Key](https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag#step_1_create_a_project).
 
-## Using feature flags metrics
+### Install the FF SDK Dependency
 
-Metrics API endpoint can be changed like this:
-
-```
-Config.builder()
-              .eventUrl("METRICS_API_EVENTS_URL")
-              .build();
-```
-
-Otherwise, the default metrics endpoint URL will be used.
-
-## Listen on events
-
-```
-client.on(Event.READY, result -> log.info("READY"));
-
-client.on(Event.CHANGED, result -> log.info("Flag changed {}", result));
-```
+The first step is to install the FF SDK as a dependency in your application using your application's dependency manager. You can use Maven, Gradle, SBT, etc. for your application.
 
-events will work even when streamEnabled is off.
+Refer to the [Harness Feature Flag Java Server SDK](https://mvnrepository.com/artifact/io.harness/ff-java-server-sdk) to identify the latest version for your build automation tool.
 
-## Connector
+This section lists dependencies for Maven and Gradle and uses the 1.1.4 version as an example:
 
-This is a new feature that allows you to create or use other connectors.
-Connector is just a proxy to your data. Currently supported connectors:
-* Harness
-* Local (used only in development)
+#### Maven
 
+Add the following Maven dependency in your project's pom.xml file:
+```pom
+<dependency>
+    <groupId>io.harness</groupId>
+    <artifactId>ff-java-server-sdk</artifactId>
+    <version>1.1.4</version>
```

Comment:
Could we use LATEST as the version here, or maybe something like `[1.0.0,2.0.0)` so that this example will work with all future releases until we make a major change?